### PR TITLE
[run_copr_build()] Move exception string from status to comment

### LINKED ISSUE #105: copr builds: create a comment instead of setting a status
if there is an error during execution, we set status on the PR with description from the exception: github doesn't allow us to put there too much text; we should instead create a comment on the PR and include the text inside while setting status to something short, e.g. "There was an error during the build process".

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #105

### CODE REVIEW

File: packit_service/worker/copr_build.py [hunk 1]
```diff
@@ -157,11 +157,11 @@ def run_copr_build(self) -> HandlerResults:
...
-            msg = f"There was an error while running a copr build. {msg_retrigger}"
-            self.project.pr_comment(self.event.pr_id, msg)
-            logger.error(f"error while running a copr build: {ex}")
-            msg = f"There was an error while running the build: {ex}"
-            r.report("failure", msg, check_name=check_name)
+            msg = f"There was an error while running a copr build: {ex}"
+            logger.error(msg)
+            self.project.pr_comment(self.event.pr_id, f"{msg}\n{msg_retrigger}")
+            # status text can't be too long, put just the exception there
+            r.report("failure", f"{ex}", check_name=check_name)
```
Comment (TomasTomecek) [inline packit_service/worker/copr_build.py]: I would put here a static text saying that the build failed and they should check the pr comment

I can easily imagine how some exception messages could make this really cryptic, e. g. keyerror

Comment (jpopelka) [reply packit_service/worker/copr_build.py]: You thumbs-upped my [comment](https://github.com/packit-service/packit-service/issues/105#issuecomment-526092170) about putting just the exception there :-)

But I see your point, what about now?

File: packit_service/worker/copr_build.py [hunk 2]
```diff
@@ -157,11 +157,12 @@ def run_copr_build(self) -> HandlerResults:
...
-            msg = f"There was an error while running a copr build. {msg_retrigger}"
-            self.project.pr_comment(self.event.pr_id, msg)
-            logger.error(f"error while running a copr build: {ex}")
-            msg = f"There was an error while running the build: {ex}"
-            r.report("failure", msg, check_name=check_name)
+            msg = f"There was an error while running a copr build: {ex}"
+            logger.error(msg)
+            self.project.pr_comment(self.event.pr_id, f"{msg}\n{msg_retrigger}")
+            r.report(
+                "failure", "Build failed, check latest comment.", check_name=check_name
```
Comment (rpitonak) [inline packit_service/worker/copr_build.py]: Huge nit:
`Build failed, check latest comment.` -> `Build failed, check the latest comment for more details.` or is it too long?

Comment (jpopelka) [reply packit_service/worker/copr_build.py]: It seems to be longer than what can be displayed.
What about a compromise? (check now)

Comment (rpitonak) [reply packit_service/worker/copr_build.py]: +1

File: packit_service/worker/copr_build.py (hunk 1 above)
Comment (TomasTomecek) [reply packit_service/worker/copr_build.py]: I thumbs up everything.

Comment (jpopelka) [reply packit_service/worker/copr_build.py]: go thumb up yourself  :+1:

**Pull request merged**